### ISSUE TITLE
Collection and Work count numbers not displaying correctly on user profile

### ISSUE THREAD

**Issue by jlhardes** [samvera/hyrax#5856]
### Descriptive summary

The user profile on the Dashboard and in the public view normally shows a number count in white font on a dark colored bubble (not sure of exact color). The number count is still there but the bubble is missing so we are seeing white font on a white background.

### Expected behavior

Not sure what the background color should be behind these numbers. Even just updating the font color for the numbers to something that contrasts enough for accessible visibility would work. There doesn't have to be a bubble of color in place.

### Actual behavior

User profile in the Dashboard:

![Screen Shot 2022-09-12 at 3 23 36 PM](https://user-images.githubusercontent.com/3440166/189738547-8ff7961f-d06b-4fb5-89e3-0463556c418d.png)

User profile in the public view:

![Screen Shot 2022-09-12 at 3 24 07 PM](https://user-images.githubusercontent.com/3440166/189738595-6c0406d6-4120-406b-aba7-2325aecb7631.png)

### Steps to reproduce the behavior

1. Log in to nurax-dev (or nurax-pg)
2. Go to Dashboard > Your Activity > Profile
3. See what looks like a missing count for Collections and Works
4. You can highlight the text on the screen with the cursor to see that the numbers are there but are white font on a white background.
5. Go to Home page for public view, click on Recently Uploaded tab, and click on Depositor's name
6. Repeat steps 3 and 4 to see that numbers are there for Works and Collections but they are not visible.

**Comment by rjkati** [comment]
Confirmed that collection and work count numbers are displaying on nurax-dev (hyrax 4.0.0.beta2)
![profile_count](https://user-images.githubusercontent.com/32644254/221025617-d13844b3-4519-46a4-a231-1c36fc8568e8.PNG)

![public_profile_count](https://user-images.githubusercontent.com/32644254/221025869-53e41815-700b-47d4-8746-c08cd2444dc1.PNG)
